sql/migrations/20210704100709_world.sql

```diff
@@ -54,6 +54,14 @@ VALUES
     ('8284', '38', '-6387.385742', '-1983.494263', '246.722427', '5000', 'take a breather'),
     ('8284', '39', '-6387.385742', '-1983.494263', '246.722427', '30000', 'death');
 
+-- Disable initial spawn of 'Singed Letter', in order to create it at the end of 'Suntara Stones'.
+UPDATE gameobject
+SET
+    spawntimesecsmin = -10,
+    spawntimesecsmax = -10
+WHERE
+    guid = '10106';
```

Comment:
Make that query be all on the same line, and don't put quotation marks around numbers.

```
UPDATE `gameobject` SET `spawntimesecsmin`=-10, `spawntimesecsmax`=-10 WHERE `guid`=10106;
```

Reply:
Done